core/src/main/java/org/mini2dx/Tappybird/Background.java

```diff
@@ -0,0 +1,25 @@
+package org.mini2dx.Tappybird;
+
+import com.badlogic.gdx.graphics.Texture;
+import org.mini2Dx.core.graphics.Graphics;
+
+import static org.mini2dx.Tappybird.TappyBirdGame.flyingspeed;
+
+public class Background extends Hazards {
+
+    Texture background = new Texture("background.png");
```

Comment:
"background.png" is a magic string and should be moved to a static final variable.

Generally it's a good idea to reuse Texture instances as much as possible. The current implementation will load a new texture and push it to the graphics card every time you make an instance.

Reply:
I implemented a flyweight pattern to stop this from happening.
I generate one instance of a class which contains the shared texture, and I pass a reference to this class where needed.